mono/mini/exceptions-amd64.c

```diff
@@ -337,7 +337,7 @@ mono_arch_get_restore_context_full (guint32 *code_size, MonoJumpInfo **ji, gbool
 	amd64_mov_reg_reg (code, AMD64_R11, AMD64_ARG_REG1, 8);
 	fregs_offset = G_STRUCT_OFFSET (MonoContext, fregs);
 	for (i = 0; i < AMD64_XMM_NREG; ++i) {
-		amd64_movsd_reg_membase (code, i, AMD64_R11, fregs_offset + (i * sizeof (MonoContextSimdReg)));
+		amd64_sse_movups_reg_membase (code, i, AMD64_R11, fregs_offset + (i * sizeof (MonoContextSimdReg)));
```

Comment:
Does `amd64_sse_movups_reg_membase` do the same thing as `amd64_movsd_reg_membase` plus more? Or do we need to call both?